src/test/java/org/folio/util/HoldingsStatusUtil.java

```diff
@@ -36,7 +36,6 @@ public static HoldingsLoadingStatus saveStatus(String credentialsId, HoldingsLoa
                                                  OffsetDateTime startedTime, Vertx vertx) {
     CompletableFuture<HoldingsLoadingStatus> future = new CompletableFuture<>();
     String query = DbUtil.prepareQuery(insertLoadingStatus(), holdingsStatusTestTable(), createPlaceholders(4));
-    status.getData().getAttributes().setStarted(startedTime.format(DateTimeFormatter.ISO_OFFSET_DATE_TIME));
```

Comment:
Remove unused import statement from 21st line, causing pipeline to fail

Reply:
Thanks! ✔️